### PR TITLE
HTCONDOR-2787 Replace "worker node" with "execution point"

### PR DESCRIPTION
**Pull request by GregThain**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [ ] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [ ] Verify that the branch destination of the PR matches the target version of the ticket
- [ ] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [ ] Check for documentation, if needed  (documentation [build logs](https://readthedocs.org/projects/htcondor/builds/))
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/admin-manual/ep-policy-configuration.rst [hunk 1]
```diff
@@ -3634,7 +3634,7 @@ on the execute directories for all the startd machines:
 
 All docker universe jobs can request either host-based networking
 or no networking at all.  The latter might be for security reasons.
-If the worker node administrator has defined additional custom docker
+If the EO administrator has defined additional custom docker
```
Comment (ColeBollig) [inline docs/admin-manual/ep-policy-configuration.rst:3637]:
```suggestion
If the EP administrator has defined additional custom docker
```

File: docs/man-pages/condor_submit.rst [hunk 2]
```diff
@@ -2228,13 +2228,13 @@ COMMANDS FOR THE DOCKER UNIVERSE
...
     a private network interface.  Some administrators may define
-    site specific docker networks on a given worker node.  When this
+    site specific docker networks on a given execution point.  When this
     is the case, additional values may be valid here.
 
  :subcom-def:`docker_pull_policy` = < always >
     if docker_pull_policy is set to *always*, when a docker universe job
-    starts on a worker node, the option "--pull always" will be passed to
-    the docker run command.  This only impacts worker nodes which already
+    starts on a execution pointer, the option "--pull always" will be passed to
```
Comment (ColeBollig) [inline docs/man-pages/condor_submit.rst:2236]:
```suggestion
    starts on a execution point, the option "--pull always" will be passed to
```

File: docs/users-manual/file-transfer.rst [hunk 3]
```diff
@@ -6,20 +6,19 @@ Submitting Jobs Without a Shared File System: HTCondor's File Transfer Mechanism
...
-what we call the executable and all the declared input files of a job.  The
-set of all files created by the job is the output sandbox.
+HTCondor works well without a shared file system between the access points and
+the execution points. The HTCondor file transfer mechanism allows the user to
+explicitly select which input files are transferred to the EP before
+the job starts. HTCondor will transfer these files, potentially delaying this
+transfer request, if starting the transfer right away would overload the access
+point.  Queueing requests like this prevents the crashes so common with
+too-busy shared file servers. These input files are placed into a scratch
+directory on the EP, which is the starting current directory of the
```
Comment (ColeBollig) [inline docs/users-manual/file-transfer.rst:16]: Should we say `sandbox directory` rather than `scratch directory` in this sentence?

Comment (GregThain) [reply docs/users-manual/file-transfer.rst:16]: I could go either way, but I think i want to differentiate "sandbox", as a set of file that can live in one of many places, vs. the "scratch directory", which is a place on the EP where a sandbox can live.

**Pull request merged**